content/docs/user-guide/how-to/tips-and-tricks.md

```diff
@@ -0,0 +1,10 @@
+# Tips and tricks for DVC Projects
+
+This guide provides general tips and tricks related to DVC, which can be
+utilized while working on a project. Using the practices listed here, you can
```

Comment:
Let's put this directly in user-guide/ please. Not a how-to. Keep it hidden from nav for now. Thanks

Reply:
Can we also extract this to the Best Practices PR please?